/**:
  ros__parameters:
    input_frame: "base_link"
    output_frame: "base_link"
    min_x: -60.0
    max_x: 60.0
    min_y: -60.0
    max_y: 60.0
    min_z: -30.0
    max_z: 50.0
    negative: False
